### ISSUE TITLE
When switching from a group chat -> new friend request, the text in the send message box is copied over.

### ISSUE THREAD

**Issue by GrayHatter** [notsecure/uTox#779]

**Comment by tsudoko** [comment]
Can't reproduce, gentoo x86_64, uTox faefe96

**Comment by GrayHatter** [comment]
@tsudoko 
```
start a group chat without sending a message but enter some text 
get a new friend request, switch to it, accept it
text with be in both
```
is this what you did?

**Comment by tsudoko** [comment]
No, I thought you meant the "add friend" screen. Can confirm, happens with 1v1 chats as well.

**Comment by GrayHatter** [comment]
might be closed by #894 @tsudoko can you check too?

**Comment by tsudoko** [comment]
Seems to be fixed
